nova_vm/src/ecmascript/syntax_directed_operations/function_definitions.rs

```diff
@@ -28,7 +29,7 @@ pub(crate) fn instantiate_ordinary_function_object(
     // FunctionDeclaration : function BindingIdentifier ( FormalParameters ) { FunctionBody }
     if let Some(id) = &function.id {
         // 1. Let name be StringValue of BindingIdentifier.
-        let _name = id.name.clone();
+        let name = id.name.clone();
```

Comment:
**nitpick:** `clone()` here is no longer relevant. (This used to be an owned string, now it's a reference.)